src/store/type.ts

```diff
@@ -107,6 +108,8 @@ export type QuasarDialog = QVueGlobals["dialog"];
 
 export type AudioStoreState = {
   characterInfos: Record<string, CharacterInfo[]>;
+  morphableTargetsInfo: Record<string, MorphableTargetsInfo>;
+  morphableTargetsCacheKey: Record<string, Array<number>>;
```

Comment:
numberになっているため何の値が入ってるか混乱しそうです。こうするとちょっとマシかも。
```suggestion
  morphableTargetsInfo: Record<string, MorphableTargetsInfo>;
  morphableTargetsCacheStyleIds: Record<string, Array<number>>;
```